data/overlayfiles/motd-manager-server-byos/etc/motd

```diff
@@ -0,0 +1,19 @@
+SUSE Manager {SUMA_VERSION} Server
+{OS_PRETTY_NAME} {ARCH} (64-bit)
+
+    ____
+  /@    ~-.
+  \/ __ .- |
+   // // @
+
+Before setting up SUSE Manager, please make sure the system's host name
+is configured to match its DNS name.
+
+To setup SUSE Manager please execute: sudo /sbin/yast2 susemanager_setup
+
+{INCLUDES}
+Documentation: https://www.suse.com/documentation/suse_manager/
+BYOS Guide: https://documentation.suse.com/suma/4.3/en/suse-manager/specialized-guides/qs-publiccloud/overview.html
```

Comment:
That link contains a hard-coded version number. It could be replaced with https://documentation.suse.com/suma/{SUMA_VERSION}/en/suse-manager/quickstart-public-cloud/qs-publiccloud-overview.html which appears to work for 4.3 and 4.2. Links into documentation tend to break over time though.